util/webhook/github-ping-event.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
Can you move this file to a `testdata` directory?

Reply:
Sure! I also moved all the other webhook event data with it and updated their tests 👍